setup.py

```diff
@@ -22,6 +23,15 @@
 SETUP_COMMANDS = {}
 
 
+def set_python_path(path):
```

Comment:
can you do a separate sync commit, so these upstream changes are not in your PR.

Reply:
Well these changes are there because the variable `docs_output_dir` has been changed from `false` to `docs`. Hence a part of this PR.

Reply:
Ok, then the sync can be a separate commit in your PR.